.gitignore

```diff
@@ -34,7 +35,7 @@ MANIFEST
 # Installer logs
 pip-log.txt
 pip-delete-this-directory.txt
-
+*.log
```

Comment:
This shouldn't be needed. I am pretty sure its already in there as a rule under the Django section (autogenerated crap already took care of this for us)